tools/pythonpkg/duckdb/experimental/spark/sql/readwriter.py

```diff
@@ -20,6 +20,47 @@ def saveAsTable(self, table_name: str) -> None:
         relation = self.dataframe.relation
         relation.create(table_name)
 
+    def parquet(self, path: str, mode: Optional[str] = None, partitionBy: Union[str, List[str], None] = None, compression: Optional[str] = None) -> None:
+        relation = self.dataframe.relation
+        if mode:
+            raise NotImplementedError
+        if partitionBy:
+            raise NotImplementedError
+
+        relation.write_parquet(path, compression=compression)
+    
+    def csv(self, path: str, mode: Optional[str] = None, compression=None, sep=None, quote=None, escape=None,
+            header=None, nullValue=None, escapeQuotes=None, quoteAll=None, dateFormat=None,
+            timestampFormat=None, ignoreLeadingWhiteSpace=None, ignoreTrailingWhiteSpace=None,
+            charToEscapeQuoteEscaping=None, encoding=None, emptyValue=None, lineSep=None):
+        if mode not in (None, "overwrite"):
+            raise NotImplementedError
+        if escapeQuotes:
+            raise NotImplementedError
+        if ignoreLeadingWhiteSpace:
+            raise NotImplementedError
+        if ignoreTrailingWhiteSpace:
+            raise NotImplementedError
+        if charToEscapeQuoteEscaping:
+            raise NotImplementedError
+        if emptyValue:
+            raise NotImplementedError
+        if lineSep:
+            raise NotImplementedError
+        relation = self.dataframe.relation
+        relation.write_csv(path,
+                        sep=sep, 
+                        na_rep=nullValue,
+                        quotechar=quote,
+                        compression=compression, 
+                        escapechar=escape,
+                        header=header if isinstance(header, bool) else header == "True",
+                        encoding=encoding,
+                        quoting=quoteAll, # ~ check this
```

Comment:
Seems you left a TODO in there?

Reply:
yes - already checked it and was happy. Thanks for noticing, I have removed this comment in the latest of this branch.